macOnly.js

```diff
@@ -1,46 +1,13 @@
 const { ipcMain } = require('electron')
 
-function doubleClick (win) {
-  let smallPosition = win.getBounds()
-  let bigPosition
-  const customSize = 300
-  const bigFactor = 2 // how much enlarge when double click video
-  const proportion = customSize * bigFactor
+function changeWindowWithDoubleClick (screenController) {
+  ipcMain.on('double-click', event => {
+    const { width } = screenController.getScreenSizeInPixels()
 
-  win.setMaximumSize(proportion, proportion)
+    const size = width <= 300 ? 'large' : 'initial'
```

Comment:
Inside the ScreenController we have the value of the currentScreenSize (https://github.com/maykbrito/mini-video-me/blob/master/src/lib/ScreenController.js#L14) so we don't need to compare this with the value in pixels if we put the `toggleWindowSize` inside the ScreenController class.

Reply:
awesome!💜